### PR TITLE
fix(CI): replace clang-9 with clang-8

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
Context: https://stackoverflow.com/questions/67537679/clang9-steady-clocknow-causing-timeout-the-monitored-command-dumped-core

I've tried to reproduce the issue locally with ubuntu 20.04 and clang-9, but it worked fine for me. So it's something wrong with some dependency in our CI. I've opened a report here: https://github.com/github/hub/issues/2765

So let's try replacing clang-9 with clang-8 and see what happens.

### CODE REVIEW

**Comment by locus313** [comment]
@FrancescoBorzi I don't think that is the correct repo to report the issue to github. I think its actually this repo it should be reported at https://github.com/actions/virtual-environments.

File: apps/ci/ci-conf.sh [hunk 1]
```diff
@@ -59,6 +59,12 @@ case $COMPILER in
     echo "CCOMPILERCXX=\"clang++-9\"" >> ./conf/config.sh
     ;;
 
+  "clang8" )
+    time sudo apt-get install -y clang-8
+    echo "CCOMPILERC=\"clang-8\"" >> ./conf/config.sh
+    echo "CCOMPILERCXX=\"clang++-8\"" >> ./conf/config.sh
+    ;;
+
```
Comment (locus313) [inline apps/ci/ci-conf.sh:67]: we probably should put clang8 before clang9

File: apps/ci/ci-install.sh [hunk 2]
```diff
@@ -64,6 +64,12 @@ case $COMPILER in
     echo "CCOMPILERCXX=\"clang++-9\"" >> ./conf/config.sh
     ;;
 
+  "clang8" )
+    time sudo apt-get install -y clang-8
+    echo "CCOMPILERC=\"clang-8\"" >> ./conf/config.sh
+    echo "CCOMPILERCXX=\"clang++-8\"" >> ./conf/config.sh
+    ;;
+
```
Comment (locus313) [inline apps/ci/ci-install.sh:72]: same here

File: apps/ci/ci-conf.sh (hunk 1 above)
Comment (FrancescoBorzi) [reply apps/ci/ci-conf.sh:61]: ah definitely yes, my bad

**Review by locus313 — APPROVED** [review]

**Comment by locus313** [comment]
@FrancescoBorzi recently found this:
https://github.com/actions/virtual-environments/issues/3235

**Pull request merged**